ui/js/component/video/internal/loading-screen.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const LoadingScreen = ({ status }) =>
+  <div className="video--loading--screen">
```

Comment:
We're using [BEM](http://getbem.com/) as our CSS naming pattern. If our top level block here is `video`, then these class names should be something like:

video__loading-screen
video__loading-screen-content
video__loading-status

`--` should only be used as a modifier, e.g. `video__loading-status--error`.